db/migrate/20160906202307_create_investments.rb

```diff
@@ -0,0 +1,13 @@
+class CreateInvestments < ActiveRecord::Migration
+  def change
+    create_table :investments do |t|
+      t.string :name
+      t.integer :cost
+      t.string :invest_type
+      t.string :long_term
+      t.string :short_term
```

Comment:
invest_type could be like stock or something else. I got that from the discussion this morning